### PR TITLE
Add rails review form

### PR DESCRIPTION
**Pull request by deniseileen**

### CODE REVIEW

File: app/assets/stylesheets/constants.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  --lunch-pale: #FEFFF9;
+  --lunch-white: #FFFFFF;
+  --lunch-black: #000000;
+  --lunch-blue: #2B59C3;
+  --lunch-green: #7CB518;
+  --lunch-tan: #FFDDA1;
+  --lunch-orange: #FFB23F;
+  --lunch-red: #FE5F55;
+  /*-alt-colors-*/
+}
```
Comment (enthusiastick) [inline app/assets/stylesheets/constants.css:17]: Fancy styling, and all of the colors in their own file? 👍

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+	private
+
+	def review_params
+		params.require(:review).permit(
+			:rating,
+			:body
+		)
+	end
+end
```
Comment (enthusiastick) [inline app/controllers/reviews_controller.rb:33]: What, uh, what happened to the indentation, here? Who is responsible for this? (╯°□°）╯︵ ┻━┻

File: app/views/reviews/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<% if @review.errors.present? %>
+	<ul>
+		<p class="error">Error!</p>
+		<% @review.errors.full_messages.each do |message| %>
+			<li>
+				<%= message %>
+			</li>
+		<% end %>
+	</ul>
+<% end %>
```
Comment (enthusiastick) [inline app/views/reviews/new.html.erb:33]: Again. (╯°□°）╯︵ ┻━┻

**Comment by enthusiastick** [comment]
You are better than this

File: app/controllers/reviews_controller.rb (hunk 2 above)
Comment (deniseileen) [reply app/controllers/reviews_controller.rb:33]: It's formatted correctly in Atom.  Any idea what's causing the formatting change or how to address it?

File: app/assets/stylesheets/constants.css (hunk 1 above)
Comment (deniseileen) [reply app/assets/stylesheets/constants.css:17]: thanks eben. would you recommend keeping all the colors in their own file?

**Pull request closed without merging**